packages/breadcrumb/components/Breadcrumb.tsx

```diff
@@ -37,6 +37,7 @@ class Breadcrumb extends React.PureComponent<BreadcrumbProps, {}> {
             className={cx(textWeight("medium"), textSize("l"), {
               [padding("left", "xs")]: i !== 0
             })}
+            data-cy="breadcrumb"
```

Comment:
If we have One > Two, then "One", ">" and "Two" all have this data-cy. Is this expected?

Reply:
That is expected since each of those is a breadcrumb. Do you (or any other reviewers) think we should exclude `data-cy` on the ">" breadcrumbs?

Reply:
I don't have a strong opinion, I just though it was a little strange. If I don't know this, I would think that the second breadcrumb would be "Two", not ">".